test/sass/mixins/spacing.spec.scss

```diff
@@ -2539,3 +2539,55 @@
   }
 
 }
+
+@include describe("@mixin inset-inline") {
+
+  @include it("generates css logical properties for left and right positioning and fallbacks") {
+
+    @include assert() {
+
+      @include output {
+        @include inset-inline($start: 10px, $end: 20px);
+      }
+
+      @include expect {
+        left: 10px;
+        right: 20px;
+
+        @supports (inset-inline-start: 10px) {
+          inset-inline-start: 10px;
+          inset-inline-end: 20px;
+        }
+
+      }
+
+    }
+
+  }
+}
+
+@include describe("@mixin inset-block") {
```

Comment:
This mixin needs additional tests to confirm that it doesn't set a property when no value is passed for that property. When passing `$start` but not `$end`, verify that `top` and `inset-block-start` are both set, but also that neither `bottom` nor `inset-block-end are set. Also needs a test for the converse, when only `$end` but not `$start` is set.